tools/flang1/flang1exe/semfunc.c

```diff
@@ -2098,8 +2098,9 @@ gen_pointer_result(int array_value, int dscptr, int nactuals,
     get_all_descriptors(arr_tmp);
     /* need to have different MIDNUM than arr_value */
     /* otherwise multiple declaration */
-    pvar = sym_get_ptr(arr_tmp);
-    MIDNUMP(arr_tmp, pvar);
+//    pvar = sym_get_ptr(arr_tmp);
+//    MIDNUMP(arr_tmp, pvar);
```

Comment:
Usually, commented out code is not left in the git repo, the git commit itself express your intent clearly and in the git history no old code gets lost.

Reply:
agree. will push a new commit after removing the commented lines.